src/helpers/keyManager.ts

```diff
@@ -77,12 +77,7 @@ export const loadPrivateKey = async ({
   password: string;
 }) => {
   const keyManager = createKeyManager();
-  let result;
-  try {
-    result = await keyManager.loadKey(id, password);
-  } catch (error) {
-    return error;
-  }
```

Comment:
This `try catch` was making returned type `Error | Key`, which was creating problems down the line. We don't really need this `try catch` here, because it's handled in `submitPaymentTransaction()` [here](https://github.com/stellar/account-viewer-v2/blob/master/src/helpers/submitPaymentTransaction.ts#L41:L45).